packages/frontapp/src/app/components/BookInfo/Buttons/ReturnBookButtons.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import { ReactComponent as Claim } from '../../../../assets/claim.svg';
+import Button from '../../Button';
+
+interface IControl {
+  isClaimed?: boolean;
+  onClaim: () => void;
+  onReturn: () => void;
+  onProlong: () => void;
+}
+
+export const ReturnBookButtons: FC<IControl> = ({
+  isClaimed = false,
+  onClaim,
+  onReturn,
+  onProlong,
+}) => {
+  if (!isClaimed)
```

Comment:
Add braces or use ternary operator.
Omitting braces in js considered error-prone